decidim-participatory_processes/app/services/decidim/participatory_processes/automate_processes_steps.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
+    # This service change the processes steps automatically.
+    class AutomateProcessesSteps
+      def change_active_step
```

Comment:
Could we move this logic to the job now that I believe this is not called anywhere directly?

I think we don't need this particular class now that it's implemented in the job but I could be also mistaken, so if there is some reason why we have a dedicated service class for this, feel free to enlighten me.

Reply:
The idea of having the logic in a service was to call it from a rake task and from the job too. But with the latest change it is no longer necessary. 

I've moved the logic to the job. Can you check it, @ahukkanen? Thank you!